getting-started/sandbox-quick-start-guide.md

```diff
@@ -0,0 +1,439 @@
+# INTRODUCTION
+
+**Cloud File Transfer (CFT)** [sandbox][glossary.md] is a test environment that is almost identical to our production environment except that it supports only internet to internet transfers.
```

Comment:
Can be a littler clearer here so that people will not misread it that CFT only  supports internet to internet transfers. So maybe something like:

"... is a test environment that is almost identical to our production environment. However, the test environment will be limited to internet to internet transfers."